src/index.html

```diff
@@ -1,14 +1,61 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./style.css" >
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="header__menu">
+          <a href="#" class="header__logo">
+            <img src="./images/logo.png" alt="logo" >
```

Comment:
[Checklist](https://github.com/Rusik-best86/layout_moyo-header/blob/develop/checklist.md):
- alt attribute should describe the image content
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/137c6759-8320-4a63-a31f-0e4527872ee9)